src/components/NetworkSwitcher/NetworkSwitcher.preset.ts

```diff
@@ -4,7 +4,13 @@ import EthereumLogo from '../../assets/svg/ethereum-logo.svg'
 import ArbitrumLogo from '../../assets/svg/arbitrum-one-logo.svg'
 import GnosisLogo from '../../assets/svg/gnosis-chain-logo.svg'
 
-export const networkOptionsPreset = [
+export const networkOptionsPreset: {
+  chainId: ChainId
+  name: string
+  logoSrc: string
+  color: string
+  tag?: string
```

Comment:
The tag should be either `string[]` or enum, your call.

Reply:
Fixed in #908